src/main/java/com/android/volley/CacheDispatcher.java

```diff
@@ -48,6 +52,9 @@
     /** Used for telling us to die. */
     private volatile boolean mQuit = false;
 
+    /** Manage list of waiting requests and de-duplicate requests with same cache key. */
+    private final WaitingRequestHandler mWaitingRequestHandler;
```

Comment:
nit: Handler is a loaded term on Android (https://developer.android.com/reference/android/os/Handler.html), so for clarity's sake it would be good to avoid using it if we're not actually extending that class.

Perhaps "WaitingRequestManager" or "WaitingRequestDispatcher"?

Reply:
Went with Manager.